web/server/vue-cli/src/components/Report/SelectReviewStatus.vue

```diff
@@ -57,6 +57,53 @@
 
       <v-card-text class="pa-0">
         <v-container>
+          <v-alert
+            v-if="reportsReviewedInSource.length"
```

Comment:
Can you please re-phrase the message when removing a review status rule?
![image](https://user-images.githubusercontent.com/6695818/162932088-a0c2e90a-4aaa-418c-8554-88637c609d56.png)

It would be good to mention that removing the review status rule will also remove the fixed at date and this way also the statistics will change.
And maybe it would be also good to highlight here that this operation cannot be undone.